React/Fabric/Mounting/ComponentViews/Text/RCTParagraphComponentView.mm

```diff
@@ -8,7 +8,10 @@
 #import "RCTParagraphComponentView.h"
 #import "RCTParagraphComponentAccessibilityProvider.h"
 
+#if !TARGET_OS_OSX // [TODO(macOS GH#774)
 #import <MobileCoreServices/UTCoreTypes.h>
+#endif // ]TODO(macOS GH#774)
```

Comment:
Do we know why iOS needs this import?

Reply:
It is used further below for `kUTTypeFlatRTFD`


https://user-images.githubusercontent.com/484044/205272886-8160e5c3-b901-490e-97fb-f8f592dd3625.mov

Reply:
We had to do this before:

https://user-images.githubusercontent.com/484044/205274242-31695147-76fb-48e5-85c3-033654661154.mov

Reply:
On macOS we get the constant for free due to some implicit header file import
<img width="1051" alt="image" src="https://user-images.githubusercontent.com/484044/205274368-eb203fc4-48b6-4878-89dd-6c00ff8d771d.png">